spring-boot/ServletReg/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
```

Comment:
Note: I don't see this ServletReg module listed as a module under the spring-boot pom.xml.
It will need to be added there in order for the module to built along with the rest of the repo.